WKData/Sources/WKData/Models/Shared/WKEditSummaryTag.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+/// Hashtags appended to edit summaries throughout the app, for identification in mediawiki history.
+public enum WKEditSummaryTag: String {
+    case articleSectionSourceEditor = "#article-section-source-editor"
+    case articleFullSourceEditor = "#article-full-source-editor"
+    case articleSelectSourceEditor = "#article-select-source-editor"
+    case talkFullSourceEditor = "#talk-full-source-editor"
+    case articleDescriptionAdd = "#article-add-desc"
+    case articleDescriptionChange = "#article-change-desc"
+    case articleHistoryDiffUndo = "#article-history-diff-undo"
+    case articleHistoryDiffRollback = "#article-history-diff-rollback"
+    case otherDiffUndo = "#other-diff-undo"
+    case otherDiffRollback = "#other-diff-rollback"
+    case watchlistDiffUndo = "#watchlist-diff-undo"
+    case watchlistDiffRollback = "#watchlist-diff-rollback"
```

Comment:
Just FYI, leaving this PR in draft until I resolve this. Capturing where they came to the diff opens up a ton of combinations when you also consider namespaces. I'm advocating that we simplify to just `diff-undo` and `diff-rollback`. Will update this PR when I have the go ahead.